### PR TITLE
Forms Automated Test - LPS-133782 | master

### PR DESCRIPTION
**Pull request by piresdm**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase [hunk 1]
```diff
@@ -278,14 +278,29 @@ definition {
...
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		FormFields.editParagraphBody(content = "Test");
+
+		FormsAdminNavigator.gotoPreviewForm();
+
+		FormFields.viewParagraphBodyContent(key_bodyName = "Test");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase:302]: @piresdm, please rename:

- `key_bodyName` to `bodyName`

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 2]
```diff
@@ -1025,6 +1025,12 @@ definition {
 			locator1 = "Form#PAGINATION_NUMBER");
 	}
 
+	macro viewParagraphBodyContent {
+		AssertElementPresent(
+			key_bodyName = "${key_bodyName}",
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:1030]: @piresdm, please follow the pattern:

```
key_bodyName = "${bodyName}"
```

**Review by natocesarrego — CHANGES_REQUESTED** [review]
Good job, @piresdm! :slightly_smiling_face:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase (hunk 1 above)
Comment (piresdm) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase]: @natocesarrego done!

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro (hunk 2 above)
Comment (piresdm) [reply portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro]: @natocesarrego done!

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**